core/db/migrate/20161012151358_opening_ledger_entries_for_store_credits.rb

```diff
@@ -0,0 +1,12 @@
+class OpeningLedgerEntriesForStoreCredits < ActiveRecord::Migration[5.0]
+  # Prevent everything from running in one giant transaction in postrgres.
+  disable_ddl_transaction!
```

Comment:
Is this necessary? I would rather have a transaction unless there is a reason not to.